CompletedProcess(args=['python', 'temp_test_case.py'], returncode=1, stdout=b'', stderr=b'F.FF\n======================================================================\nFAIL: test0 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen/temp_test_case.py", line 12, in test0\n    self.assertEqual(patched_source(input_0), original_source(input_0))\nAssertionError: Lists differ: [\'hello\'] != [\'Hello\']\n\nFirst differing element 0:\n\'hello\'\n\'Hello\'\n\n- [\'hello\']\n?   ^\n\n+ [\'Hello\']\n?   ^\n\n\n======================================================================\nFAIL: test2 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen/temp_test_case.py", line 24, in test2\n    self.assertEqual(patched_source(input_2), original_source(input_2))\nAssertionError: Lists differ: [\'hello\'] != [\'Hello\']\n\nFirst differing element 0:\n\'hello\'\n\'Hello\'\n\n- [\'hello\']\n?   ^\n\n+ [\'Hello\']\n?   ^\n\n\n======================================================================\nFAIL: test3 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen/temp_test_case.py", line 30, in test3\n    self.assertEqual(patched_source(input_3), original_source(input_3))\nAssertionError: Lists differ: [\'helloworld\'] != [\'Helloworld\']\n\nFirst differing element 0:\n\'helloworld\'\n\'Helloworld\'\n\n- [\'helloworld\']\n?   ^\n\n+ [\'Helloworld\']\n?   ^\n\n\n----------------------------------------------------------------------\nRan 4 tests in 0.001s\n\nFAILED (failures=3)\n')